spec/section_spec.rb

```diff
@@ -49,6 +118,38 @@ class PageWithAnonymousSection < SitePrism::Page
   let(:locator) { instance_double('Capybara::Node::Element') }
   let(:section_with_block) { SitePrism::Section.new(Page.new, locator) { 1 + 1 } }
 
+  describe '#default_search_arguments' do
+    let(:base_section) { Class.new(SitePrism::Section) }
+    let(:section)      { Class.new(base_section) }
+    let(:arguments)    { section.default_search_arguments }
+
+    it 'should be [] by default' do
+      expect(arguments).to eql []
+    end
+
+    it { expect(section).to respond_to(:set_default_search_arguments) }
+
+    context 'if default_search_arguments are set' do
+      before do
+        section.set_default_search_arguments '//body', text: 'Body'
+      end
+
+      it 'should return default search arguments' do
+        expect(arguments).to eql(['//body', text: 'Body'])
+      end
+
+      context 'if base class default_search_arguments are set' do
+        before do
+          base_section.set_default_search_arguments :xpath
```

Comment:
This seems erroneous, doesn't this just imply that you've set a base searching path of ... nothing? I know you're in a WIP so ignore this if irrelevant

Reply:
the idea was to allow to concatenate search arguments. something like
```ruby
#base class
set_default_search_arguments :xpath, '//html'

#child class
set_default_search_arguments text: 'Name'

# will produce search arguments like: 
# :xpath, '//html', text: 'Name'
```

But I'm going to change it as you recommended in PR comment to always overrride with child arguments if they present